### PR TITLE
Add `analysis` and `dispersion` to `plasmapy.__init__`

### PR DESCRIPTION
**Pull request by namurphy**
This PR adds the `analysis` and `dispersion` subpackages to `plasmapy.__init__`, including in the `__all__` dunder.

### CODE REVIEW

File: changelog/1512.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The `~plasmapy.analysis` and `~plasmapy.dispersion` subpackages are now
+directly imported in `plasmapy.__init__`.
```
Comment (rocco8773) [inline changelog/1512.bugfix.rst:2]: Wrong tense...

```suggestion
Exposed the `~plasmapy.analysis` and `~plasmapy.dispersion` subpackages to the
`plasmapy` namespace..
```

Comment (rocco8773) [reply changelog/1512.bugfix.rst:2]: Good catch on this.  I'm at a loss for how this has been missed for so long.

Comment (namurphy) [reply changelog/1512.bugfix.rst:2]: It is indeed import-ant.  😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹

Comment (rocco8773) [reply changelog/1512.bugfix.rst]: You made the rules, I'm only just enforcing them. 😆

**Comment by rocco8773** [comment]
I'm a little puzzled right now since your changelog entry is not rendering on the changellog page in the built documentation.  I currently can't see why this would happen.

**Comment by rocco8773** [comment]
Never mind, the entry just showed up. ...And I was about to comment on the extra period. 🤣

**Review by rocco8773 — APPROVED** [review]

**Pull request merged**